.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on: push
+
+jobs:
+    test:
+        name: Test
+
+        runs-on: ubuntu-18.04
+
+        strategy:
+            matrix:
+                php-version: ['7.2', '7.4']
+            fail-fast: false
+
+        steps:
+            # Setup
+            -   name: Checkout
+                uses: actions/checkout@v2
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php-version }}
+                    coverage: pcov
+                    tools: cs2pr, prestissimo
+                    extensions: bcmath, intl
+            -   name: Cache Dependencies
+                uses: actions/cache@v1
+                with:
+                    path: ~/.composer/cache
+                    key: php${{ matrix.php-version }}-composer-${{ hashFiles('**/composer.lock') }}
+                    restore-keys: |
+                        php${{ matrix.php-version }}-composer-
+            -   name: Install Dependencies
+                run: composer install --no-ansi --no-interaction --no-progress --no-suggest
+
+            # Test
+#            -   name: Codestyle
```

Comment:
you can also add this test?
or does the repo still contain violations?

Reply:
Unfortunately it contains loads of violations. Most of them are typehints, which can't be resolved without breaking the monolith. I intend to resolve this in a new major version that can be used in new services.